generators/Exercises/Proverb.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Generators.Output;
+
+namespace Generators.Exercises
+{
+    public class Proverb : GeneratorExercise
+    {
+        protected override string RenderTestMethodBodyAssert(TestMethodBody testMethodBody)
+        {
+            if  (testMethodBody.CanonicalDataCase.Properties["input"]["strings"] as string[] == null)
+            {
+                return TemplateRenderer.RenderInline("Assert.Empty(Proverb.Recite(new string[] { }));", new { });
```

Comment:
This shouldn't be necessary if I recall correctly, but I assume you wouldn't use it otherwise I guess. What happens when you omit these lines?

Reply:
Compile time error for the first test method.